Integrations/integration-ReversingLabsTitaniumCloud.yml

```diff
@@ -0,0 +1,221 @@
+description: 'ReversingLabs Data provides malware status of the sample. Status can
+  be: malicious, suspicious, known and unknown. Service supports single or bulk queries,
+  with extended option for response data (trust factor, threat level, ...).'
+configuration:
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Base URL
+  name: base
+  defaultvalue: https://ticloud-cdn-api.reversinglabs.com
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    import base64
+    import requests
+    from requests.auth import HTTPBasicAuth
+    import re
+    import os
+
+    base_url = demisto.params()['base']
+    user = demisto.params()['credentials']['identifier']
+    password = demisto.params()['credentials']['password']
+    auth = HTTPBasicAuth(user,password)
+
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+
+    def rldata(hash_value):
+        type_dict = {
+            32: {
+                'type':'md5',
+                'regex':r'([a-fA-F\d]{32})'
+            },
+            40: {
+                'type':'sha1',
+                'regex':r'([a-fA-F\d]{40})'
+            },
+            64: {
+                'type':'sha256',
+                'regex':r'([a-fA-F\d]{64})'
+            }
+        }
+
+        if len(hash_value) not in type_dict.keys():
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : "Not Valid Hash"})
+            sys.exit(0)
+
+        if not re.match(type_dict[len(hash_value)]['regex'],hash_value):
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : "Not Valid Hash1" })
```

Comment:
`Hash1` is a bit confusing. maybe not valid by regex, has illegal chars?

Reply:
Typo